### DISCUSSION TITLE
default user created with 'admin@asciinema.org' with asciinema server

### DISCUSSION THREAD

**Discussion by plasmium1** [asciinema/discussions#250] [Q&A]
There is a default user created with 'admin@asciinema.org' in psqldb. Is it ok to delete this default user? not sure about the purpose of creating this default user, and which command to use to disable/delete this user.

**Comment by ku1ik** [comment]
The default user is there just so we have someone to attach the intro recording to. Other than that it's not needed for anything, and can be deleted.

To do that, attach to the container and enter the Elixir shell:

```sh
docker compose exec asciinema bin/asciinema remote
```

Once in the Elixir shell delete the user with:

```elixir
Asciinema.delete_user(Accounts.find_user("admin@asciinema.org"))
```

**Comment by ku1ik** [comment]
That intro recording and the initial asciinema user is no longer added automatically on new installations since the latest server release.

New release includes a proper [admin panel](https://docs.asciinema.org/manual/server/self-hosting/admin/) so you can delete any user there.
